webapp/sources/api-doc/paths/info/section-id.yml

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2021 Normation SAS
+get:
+  summary: Get information on endpoint in a section
+  description: Get all endpoints in the given section with their supported version.
+  operationId: apiSubInformations
```

Comment:
Information cannot be plural but I guess it's hardcoded in the API

Reply:
yes it is